lab-david/lib/server-toggle.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const debug = require('debug')('cfgram:server-toggle');
+const mongoose = require('mongoose');
+
+module.exports = exports = {};
+
+exports.serverOn = function(server ,done){
```

Comment:
you should look into making server toggles that use promises, you can write some clean stuff with em. That way you can have hooks that literally read as:
```
beforeAll(server.start);
afterAll(server.stop);
```
cleannnnn